### DISCUSSION TITLE
Why is the docker version always behind all the other platforms?

### DISCUSSION THREAD

**Discussion by TheBartoMan** [AdguardTeam/AdGuardHome#5961] [Q&A]
I have the latest docker image, i use watchtower which also checks daily.

However the docker version seems to stay not get updated like the other and will be behind on the version for a long time.

**Comment by Freebase394** [comment]
Because Docker versions have many more particularities than x64 or x86 environments! 
But I have run Docker from AdguardHome to the latest version and I think it is not too late! 
Which docker image are you using ?

**Comment by TheBartoMan** [comment]
It is the the official adguard:latest docker release.

My pfsense Linux install is newer and has been for a while.

Googlemail
________________________________

**Comment by jeromel1980** [comment]
Hi
i've the same question

in the topic docker adguard home updater #6411

hi
on my synology nas, i run adguardhome docker container
in docker, on the dashboard i can see that
run command : --no-check-update
but from the registry, i pick the first choice "adguard/adguardhome"
then i chosse the latest image
 
and then my adguardhome interface looks like
 
[adguardhome.docx](https://github.com/AdguardTeam/AdGuardHome/files/13333823/adguardhome.docx)

is the update ofadguardhome automatic or it's normal because i choose "latest" ?
thanks
